pieces/meilisearch_with_cli/README.md

```diff
@@ -0,0 +1,384 @@
+# Interact with Meilisearch though a CLI
+
+Update settings, index data, and search using [Meilisearch](https://www.meilisearch.com/) with a CLI.
+
+## Goal
+
+Meilisearch is a fast, typo-tolerate search engine with a focus on being easy to setup and use. In
+Meilisearch, indexes are created to store documents and their associated settings. These indexes are
+simlilar to tables in relational databases or collections in document databases.
+
+In this example we will create and update the settings for an index, send documents to the server for
+indexing, and preform searaches all through a CLI.
+
+For data we will use the [wine reviews dataset from Kaggle](https://www.kaggle.com/datasets/zynicide/wine-reviews).
+The dataset is in the following format:
+
+```json
```

Comment:
```suggestion
```

Reply:
There's a rendering error because `null` isn't valid JSON for the syntax highlighter. I recommend leaving the `json` part out.